lib/rubocop/rspec/top_level_group.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module RSpec
+    # Helper methods for top level example group cops
+    module TopLevelGroup
+      extend NodePattern::Macros
+
+      def_node_matcher :example_or_shared_group?,
+                       (
+                         Language::ExampleGroups::ALL +
+                         Language::SharedGroups::ALL
```

Comment:
I think `Language` is included (or you can include it if it's not), and you may skip this part of the namespace, and shrink it into less lines.